functests/test_suite_test.go

```diff
@@ -17,7 +17,8 @@ import (
 	_ "github.com/openshift-kni/cnf-features-deploy/functests/ptp"  // this is needed otherwise the ptp test won't be executed
 	"github.com/openshift-kni/cnf-features-deploy/functests/sctp"
 	_ "github.com/openshift-kni/cnf-features-deploy/functests/sctp" // this is needed otherwise the sctp test won't be executed
-
+	"github.com/openshift-kni/cnf-features-deploy/functests/xt_u32"
+	_ "github.com/openshift-kni/cnf-features-deploy/functests/xt_u32"                // this is needed otherwise the xt_u32 test won't be executed
```

Comment:
The line above is enough to import the tests. This is needed only if you are not importing any symbol from the package.